collections/_sdk/effects/layout.md

```diff
@@ -255,6 +250,28 @@ The `LaunchModalEffect` class has the following properties:
   - `RIGHT_CHART_PANE_LARGE`: Like above, but a bit wider.
 - **title**: A string containing the title of the modal and will be displayed when minimized. Defaults to `Untitled`
 
+### Closing Modals from Applications
+
+When building applications with the Canvas SDK, you may encounter scenarios where you need to programmatically dismiss modals. This can be particularly useful in automated testing or when creating user flows that require closing modals based on certain conditions.When building applications with the Canvas SDK, you may encounter scenarios where you need to programmatically dismiss modals. This can be particularly useful in automated testing or when creating user flows that require closing modals based on certain conditions.
```

Comment:
```suggestion
When building applications with the Canvas SDK, you may encounter scenarios where you need to programmatically dismiss modals. This can be particularly useful in automated testing or when creating user flows that require closing modals based on certain conditions.
```